.jenkins/Jenkinsfile

```diff
@@ -22,517 +22,447 @@
 // Validate/lint this file using the following command
 // `curl -X POST  -F "jenkinsfile=<.jenkins/Jenkinsfile" https://ci-cassandra.apache.org/pipeline-model-converter/validate`
 
+def jdksSupported = ["11", "17"] 
+def archsSupported = ["amd64", "arm64"]
+def pythonsSupported = ["3.7", "3.8", "3.11"] 
+def jdkDefault = "11"
+def pythonDefault = "3.8"
+def stageResults = [:]
+def splits = 10
```

Comment:
splits differ for each test type

Reply:
It's the "generic" split count, they can be specified explicitly per stage in the `stepMap`